app/views/developers/developers.scala.html

```diff
@@ -12,19 +12,37 @@
 
 <article class="content__body full-width">
 
+    <style>
+        .developer-list tr {
+          word-break: break-all;
+        }
+
+        .input-select {
+           margin: 0;
+        }
+
+        .input-select--large {
+          width: 85%;
+        }
+
+        .developer-status {
+          min-width: 150px;
+        }
+    </style>
+
```

Comment:
yes, but there aren't any for this project. Have an outstanding question with EdF about best way to go forward with custom CSS across projects - obv. touches this and RAML project